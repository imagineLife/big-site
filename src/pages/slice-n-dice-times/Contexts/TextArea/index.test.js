import React, { useContext } from 'react';
import { TextAreaContext, TextAreaProvider } from './';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('TextAreaContext', () => {
  const ChildBox = () => {
    const ctxVals = useContext(TextAreaContext)
    return <div id="box">Child Box</div>
  };

  const init = mount(
    <TextAreaProvider>
      <ChildBox />
    </TextAreaProvider>
  );

  it('matches snapshot', () => {
    expect(toJson(init)).toMatchSnapshot()
  })
})