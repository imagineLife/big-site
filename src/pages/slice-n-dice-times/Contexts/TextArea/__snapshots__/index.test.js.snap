// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAreaContext matches snapshot 1`] = `
<TextAreaProvider>
  <ChildBox>
    <div
      id="box"
    >
      Child Box
    </div>
  </ChildBox>
</TextAreaProvider>
`;
